charts/esphome/2.0.0/questions.yaml

```diff
@@ -0,0 +1,406 @@
+                        type: int
+                        default: 6052
+                        editable: false
+                        hidden: true
+                    - variable: targetport
+                      label: "Internal Service port"
+                      description: "When connecting internally to this App, you'll need this port"
+                      schema:
+                        type: int
+                        default: 6052
+                        editable: false
+                        hidden: true
+                    - variable: nodePort
+                      label: "(optional) host nodePort to expose to"
+                      description: "only get used when nodePort is selected"
+                      schema:
+                        type: int
+                        min: 9000
+                        max: 65535
+                        default: 36052
+                        required: true
+
+  # Configure app volumes
+  - variable: appVolumeMounts
+    group: "Storage and Devices"
+    label: ""
+    schema:
+      type: dict
+      attrs:
+        # Config ------------------------
+        - variable: config
+          label: "Config Volume"
+          schema:
+            type: dict
+            $ref:
+              - "normalize/ixVolume"
+            attrs:
+              - variable: enabled
+                label: "Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+                  hidden: true
+                  editable: false
+              - variable: setPermissions
+                label: "Automatic Permissions"
+                description: "Automatically set permissions on install"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: false
+              - variable: emptyDir
+                label: "emptyDir"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+                  editable: false
+              - variable: datasetName
+                label: "Name"
+                schema:
+                  type: string
+                  default: "config"
+                  editable: false
+              - variable: mountPath
+                label: "Mount path inside the container"
+                schema:
+                  type: path
+                  default: "/config"
+                  editable: false
+              - variable: hostPathEnabled
+                label: "Config - Enable Host Path"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "Config Host Path"
+                      schema:
+                        type: hostpath
+                        required: true
+
```

Comment:
Workaround permission issue when running as non-root user by creating a mount for `/.platformio`
- this is a work around so it should remain hidden from view
- contains only bare minimum needed to work correctly
```suggestion
        # platformio ------------------------
        - variable: platformio
          label: "platformio"
          schema:
            hidden: true
            type: dict
            $ref:
              - "normalize/ixVolume"
            attrs:
              - variable: enabled
                label: "Enabled"
                schema:
                  type: boolean
                  default: true
              - variable: setPermissions
                label: "Automatic Permissions"
                schema:
                  type: boolean
                  default: true
              - variable: emptyDir
                label: "Empty Dir"
                schema:
                  type: boolean
                  default: false
              - variable: datasetName
                label: "Dataset Name"
                schema:
                  type: string
                  default: "platformio"
              - variable: mountPath
                label: "Mount Path"
                schema:
                  type: path
                  default: "/.platformio"

```

Reply:
Also confirmed  `setPermissions` is (still) working correctly
```
troy@scale:~$ sudo ls -al /mnt/ssd/ix-applications/releases/esp-202103151910/volumes/ix_volumes/
total 18
drwxr-xr-x 4 root root 4 Mar 15 19:12 .
drwxr-xr-x 3 root root 3 Mar 15 19:12 ..
drwxr-xr-x 4 apps apps 6 Mar 15 19:13 config
drwxr-xr-x 6 apps apps 7 Mar 15 19:15 platformio

troy@scale:~$ sudo ls -al /mnt/ssd/ix-applications/releases/esp-202103151910/volumes/ix_volumes/config/
total 27
drwxr-xr-x 4 apps apps   6 Mar 15 19:13 .
drwxr-xr-x 4 root root   4 Mar 15 19:12 ..
drwxr-xr-x 2 apps apps   3 Mar 15 19:13 .esphome
-rw-r--r-- 1 apps apps 233 Mar 15 19:13 .gitignore
drwxr-xr-x 6 apps apps   7 Mar 15 19:14 test8266
-rw-r--r-- 1 apps apps 342 Mar 15 19:13 test8266.yaml

troy@scale:~$ sudo ls -al /mnt/ssd/ix-applications/releases/esp-202103151910/volumes/ix_volumes/platformio/
total 32
drwxr-xr-x  6 apps apps   7 Mar 15 19:15 .
drwxr-xr-x  4 root root   4 Mar 15 19:12 ..
drwxr-xr-x 20 apps apps  21 Mar 15 19:14 .cache
-rw-r--r--  1 apps apps 186 Mar 15 19:15 appstate.json
drwxr-xr-x  2 apps apps   2 Mar 15 19:14 lib
drwxr-xr-x  7 apps apps   7 Mar 15 19:14 packages
drwxr-xr-x  3 apps apps   3 Mar 15 19:13 platforms

```